packages/web-frontend/cypress/scripts/generateConfig/generateReportConfig.js

```diff
@@ -69,7 +100,7 @@ const selectOrgUnitCode = async (database, orgUnitCodes, entityConditions) => {
 };
 
 const selectUrlParams = async (database, report, dashboardGroups) => {
-  const { viewJson } = report;
+  const viewJson = report.viewJson || {};
```

Comment:
In case you're interested, the following is also possible, though I find it unclear and prefer your syntax
```suggestion
  const { viewJson = {} } = report;
```

Reply:
I am familiar with this syntax (and I generally like it 😃 ) I didn't go this way here, because `viewJson` may be `null`

Reply:
Ah true!